sapp/ui/frontend/src/HumanReadable.js

```diff
@@ -73,7 +73,8 @@ function makeDalvikClassHumanReadable(input: string): string {
   }
 
   const split = input.split('/');
-  return split[split.length - 1].slice(0, -1);
+  const last = split[split.length - 1];
```

Comment:
`split.pop()` seems simpler since split isn't used anymore.

Reply:
okay let me use split.pop() instead

Reply:
But thinking about this 
what if there is a need to use split later (maybe in the future)
@abishekvashok

Reply:
Let's wait for others' opinions. Just my suggestion :)